src/test/java/com/google/api/codegen/testdata/nodejs_doc_message_library.baseline

```diff
@@ -756,21 +756,69 @@ var Book = {
  * @property {Object} field2
  *   This object should have the same structure as [SomeMessage2]{@link SomeMessage2}
  *
+ * @property {number} alignment
+ *   The number should be among the values of [Alignment]{@link Alignment}
```

Comment:
@jmuk What exactly is the bug for NodeJS? It sounds like from your description that the message type namespacing keeps the different enum types cleanly separated.